### PR TITLE
PORTALS-1668

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/containers/home_page/goals/Goals.tsx [hunk 1]
```diff
@@ -144,6 +144,7 @@ export default function (props: GoalsProps) {
           summary,
           link,
           asset,
+          token,
```
Comment (jay-hodgson) [inline src/lib/containers/home_page/goals/Goals.tsx:147]: This is the main bug fix, the rest is to convert old component to use react hooks instead.

File: src/lib/containers/QueryCount.tsx [hunk 2]
```diff
@@ -14,82 +14,71 @@ export type QueryCountProps = {
...
+          entityId,
+          partMask: SynapseConstants.BUNDLE_MASK_QUERY_COUNT,
+        }
+        const newIsCalculatingQueryCountForSql = {
+          ...isCalculatingQueryCountForSql,  
+        }
+        newIsCalculatingQueryCountForSql[`${sql}-${token}`] = true
+        setIsCalculatingQueryCountForSql(newIsCalculatingQueryCountForSql)
+        debugger
+        SynapseClient.getQueryTableResults(request, token).then(data => {
```
Comment (emmanmills) [inline src/lib/containers/QueryCount.tsx:56]: We should delete the debugger statement.

File: src/lib/containers/QueryCount.tsx [hunk 3]
```diff
@@ -14,82 +14,71 @@ export type QueryCountProps = {
...
+          debugger
```
Comment (emmanmills) [inline src/lib/containers/QueryCount.tsx:57]: Same here, we should remove the debugger statement.

File: src/lib/containers/home_page/goals/Goals.tsx (hunk 1 above)
Comment (emmanmills) [reply src/lib/containers/home_page/goals/Goals.tsx:147]: Got it! Thanks.

**Review by emmanmills — CHANGES_REQUESTED** [review]
Looks good, just couple small changes. Thank you!

File: src/lib/containers/QueryCount.tsx (hunk 2 above)
Comment (jay-hodgson) [reply src/lib/containers/QueryCount.tsx:55]: Oops, removed.

File: src/lib/containers/QueryCount.tsx (hunk 3 above)
Comment (jay-hodgson) [reply src/lib/containers/QueryCount.tsx]: Also removed.

**Review by emmanmills — APPROVED** [review]
Looks good!

**Pull request merged**